transport-native-kqueue/pom.xml

```diff
@@ -110,7 +110,7 @@
                       <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
                     </manifest>
                     <manifestEntries>
-                      <Bundle-NativeCode>META-INF/native/libnetty_transport_native_kqueue_${os.detected.arch}.jnilib; osname=darwin, processor=${os.detected.arch}"</Bundle-NativeCode>
+                      <Bundle-NativeCode>META-INF/native/libnetty_transport_native_kqueue_${os.detected.arch}.jnilib; osname=MacOS, processor=${os.detected.arch}"</Bundle-NativeCode>
```

Comment:
MacOSX

Reply:
@ejona86 isn't both valid ?
https://www.osgi.org/developer/specifications/reference/#ReferenceOS

Reply:
It looks like MacOS was like OS 9, the thing that is not based on Unix. I'd have expected them to be aliases if they referred to the same thing, not separate rows.

Reply:
hmm.. ok let me use macosx just to be safe :)